en/ros/README.md

```diff
@@ -26,7 +26,7 @@ PX4 supports both the "original" ROS and ROS 2, with the following configuration
 
 Unveiled at the [PX4 Developer Summit 2020](https://www.youtube.com/watch?v=lZ8crGI16qA) (and [ROS World 2020](https://www.youtube.com/watch?v=8XRkzHqQSf0)), the PX4 Development team, announced the plans to support microROS.
 
-* microRTPS:  microRTPS bridge with Fast DDS (The current stable ROS 2 interface)
-* micro XRCE-DDS: DDS on PX4 (Next step!)
-* micro ROS: ROS 2 on the PX4 (Our Target!)
+* microRTPS:  microRTPS bridge with Fast DDS (The current stable ROS 2 interface) 
+* micro XRCE-DDS: DDS on PX4 (Next step!) 
+* micro ROS: ROS 2 on the PX4 (Our Target!) 
```

Comment:
We need to update this to explain the current state - as I understand it, this is all done.

Reply:
@TSC21 I'm ramping up on this again. So to be clear, we have XRCE-DDS client library and microROS client library on PX4 right. What we don't yet do is have any code running on the FC that directly calls the microROS library - i.e. no "ROS executors".
1. Correct?
2. But in theory we could start writing ROS nodes on PX4 now?

Reply:
@hamishwillee I'll try to answer:
1. No, there is the XRCE-DDS client library but no microROS client library yet
2. For the above reason, I guess not yet